doc/classes/InputEventJoypadMotion.xml

```diff
@@ -41,10 +42,10 @@
 	</methods>
 	<members>
 		<member name="axis" type="int" setter="set_axis" getter="get_axis">
-			Joypad axis identifier, one of the JOY_AXIS_* constants in [@Global Scope].
+			Axis identifier, one of the JOY_AXIS_* constants in [@Global Scope].
 		</member>
 		<member name="axis_value" type="float" setter="set_axis_value" getter="get_axis_value">
-			Position of the axis, ranging from -1.0 to 1.0. A value of 0 means that the axis is in its neutral position.
+			Axis position, ranging from -1.0 to 1.0. A value of 0 means that the axis is in its neutral position.
```

Comment:
Current position of the joystick on the given axis. The value ranges from [code]-1.0[/code] to [code]1.0[/code]. A value of [code]0[/code] means the axis is in its resting position.